vsintegration/src/FSharp.Editor/LanguageService.fs

```diff
@@ -55,12 +55,27 @@ type internal FSharpLanguageService(package : FSharpPackage) =
         else
             None
 
+    static member UpdateOptions(projectId: ProjectId, options: FSharpProjectOptions) =
+        optionsCache.[projectId] <- options
```

Comment:
If these are source file paths, should we be using an `OrdinalIgnoreCase` comparer for the `HashSet`s?